packages/sdk/src/protocols/execution.ts

```diff
@@ -4,7 +4,7 @@
  */
 
 import { Protocol, ServerPreprocessing } from '.';
-import { ActionEvent, CollisionEvent, Context } from '..';
+import { ActionEvent, CollisionEvent, Context, WebSocket } from '..';

```

Comment:
This is not a safe assumption. Not all connections are WebSockets.

Reply:
No assumption is made, this is just an import statement. Further down in both adapters, I ask `if (this.conn instanceof WebSocket)`. Since multipeer connects to sessions via the WebSocket adapter, it all irons out in the end.